tensorboard/context.py

```diff
@@ -37,9 +37,20 @@ class RequestContext(object):
       x_forwarded_for: A tuple of `ipaddress.IPv4Address` or `ipaddress.IPv6Address`,
         which may be empty but is never None. This should be parsed value of X-Forwarded-For
         HTTP header from the request.
+      client_feature_flags: A dict of string to arbitrary type. These represent
+        feature flag key/value pairs sent by the client application. Usage of
+        client_feature_flags should know the name of the feature flag key and
+        should know its value type (typically a boolean but sometimes an array
```

Comment:
IMO we should suggest here that consumers of this field validate that the value has the expected type, rather than just assuming it is correct, since all of this data ultimately comes from the request header in the user's control so we should try not to assume it's always valid.

Reply:
Done.